File: API/RESTRODBACCESS/ResponseModel/OrderCartItemResponseModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RESTRODBACCESS.ResponseModel
{
    public class OrderCartItemResponseModel
    {
        public int billId { get; set; }
        public decimal billAmount { get; set; }
        public string billDate { get; set; }
    }
}

File: APP/app/src/main/java/a/m/restaurant_automation/responseModel/OrderCartItemResponseModel.java
package a.m.restaurant_automation.responseModel;

public class OrderCartItemResponseModel {

    private int billId;
    private double billAmount;
    private String billDate;

    public int getBillId() {
        return billId;
    }

    public void setBillId(int billId) {
        this.billId = billId;
    }

    public double getBillAmount() {
        return billAmount;
    }

    public void setBillAmount(double billAmount) {
        this.billAmount = billAmount;
    }

    public String getBillDate() {
        return billDate;
    }

    public void setBillDate(String billDate) {
        this.billDate = billDate;
    }



}
